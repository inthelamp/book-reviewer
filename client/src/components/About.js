import { Link } from 'react-router-dom'

/**
 * 
 * Presenting information about this site
 */
const About = () => {
    return (
        <div>
            <h4>Version 0.1.0</h4>
            <h4>License: <a href='https://opensource.org/licenses/MIT' target='_blank' rel='noreferrer'>MIT</a></h4>
            <h4>Contributor: <a href='https://github.com/inthelamp' target='_blank' rel='noreferrer'>IntheLamp</a></h4>
            <Link to='/'>Go Back</Link>
        </div>
    )
}

export default About
