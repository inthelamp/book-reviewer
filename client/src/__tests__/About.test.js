import renderer from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import About from '../components/About'

it('Component: About rendering correctly', () => {  
    const tree = renderer
      .create(<Router><About /></Router>)
      .toJSON();
    expect(tree).toMatchSnapshot()
})