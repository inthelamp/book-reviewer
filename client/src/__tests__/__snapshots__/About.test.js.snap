// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: About rendering correctly 1`] = `
<div>
  <h4>
    Version 0.1.0
  </h4>
  <h4>
    License: 
    <a
      href="https://opensource.org/licenses/MIT"
      rel="noreferrer"
      target="_blank"
    >
      MIT
    </a>
  </h4>
  <h4>
    Contributor: 
    <a
      href="https://github.com/inthelamp"
      rel="noreferrer"
      target="_blank"
    >
      IntheLamp
    </a>
  </h4>
  <a
    href="/"
    onClick={[Function]}
  >
    Go Back
  </a>
</div>
`;
